utils/test_utils/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "test_utils"
+version = "0.1.0"
+authors = ["ChainSafe Systems <info@chainsafe.io>"]
+edition = "2018"
```

Comment:
Would be great if all of this logic is separated by a feature that is only used in dev-dependencies. By default when we compile, or run any tests, this will have to be compiled